proto/decentraland/sdk/components/ui_dropdown.proto

```diff
@@ -0,0 +1,26 @@
+﻿syntax = "proto3";
+
+package decentraland.sdk.components;
+
+import "decentraland/common/colors.proto";
+import "decentraland/sdk/components/common/id.proto";
+import "decentraland/sdk/components/common/texts.proto";
+
+option (common.ecs_component_id) = 1094;
+
+message PBUiDropdown {
+  bool accept_empty = 1;
+  optional string empty_label = 2;
+  repeated string options = 3;
+  int32 selected_index = 4; // default=-1 when accept_empty==true; default=0 when accept_empty==false
+  bool disabled = 5;
+  optional decentraland.common.Color4 color = 6; // default=(0.0,0.0,0.0,1.0)
+  
+  optional common.TextAlignMode text_align = 10; // default='center'
+  optional common.Font font = 11; // default=0
+  optional int32 font_size = 12; // default=10
+}
+
+message PBUiDropdownResult {
```

Comment:
If this is another component, it has to be in another file with its component ID.